src/classStory.test.js

```diff
@@ -0,0 +1,22 @@
+// const Story = require("./classStory");
+import Story from './classStory';
+
+describe("classStory", () => {
+  const mockStoryData = [
+    {
+      title: 'One Sentence Wonder',
+      inputWords: {Adjective: '', Name: '', Verb: ''},
+      text: ['This is a ', ' story called ', ' and it ', '!']
+    }]  
+  const mockUserInput = {Adjective: 'reacty', Name: 'test mad lib', Verb: 'passes'}
+  const newStory = new Story(mockUserInput, mockStoryData.title);
+
+  test('it creates an instance of Story', () => {
+    expect(newStory instanceof Story).toBe(true)
+  });
+  
+  //Need help with how to have the getText function go to the storyData file for data
+  // test('it generates completed story', () => {
```

Comment:
It's able to access `storyData` but a story with the mock title you're using doesn't exist in that file.